web-app/js/portal/filter/BaseFilterPanel.js

```diff
@@ -84,6 +84,14 @@ Portal.filter.BaseFilterPanel = Ext.extend(Ext.Panel, {
     },
 
     _setExistingFilters: function() {
+    },
+
+    needsFilterRange: function() {
```

Comment:
I see this the other way around. We currently have one usage of filter possible values - the UI. So we ask the UI "Do you need to know the range of values?". If the answer is 'no' then we don't need to bother getting them. If we had some other component that might need the possible values then we could ask that, too. That way we only fetch the possible values (which is an expensive operation) if something needs them.